packaging/fedora/snapd.spec

```
@@ -887,7 +887,7 @@ make -C data -k check
 %doc cmd/snap-confine/PORTING
 %license COPYING
 %dir %{_libexecdir}/snapd
-%caps(cap_dac_override,cap_dac_read_search,cap_sys_admin,cap_sys_chroot,cap_chown,cap_fowner,cap_sys_ptrace=p) %{_libexecdir}/snapd/snap-confine
+%caps(cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_setgid,cap_setuid,cap_sys_chroot,cap_sys_ptrace,cap_sys_admin=p) %{_libexecdir}/snapd/snap-confine
```

I would not add those caps here, there's a very slim chance anyone would try to run a LXD fedora container on an old v1 host and get it to boot in the first place:

```suggestion
%caps(cap_chown,cap_dac_override,cap_dac_read_search,cap_fowner,cap_sys_chroot,cap_sys_ptrace,cap_sys_admin=p) %{_libexecdir}/snapd/snap-confine
```

---

this still needs addressing

---

I will push an update with all the comments and this change shortly.